<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_TM_mangae</class>
 <widget class="QWidget" name="Widget_TM_mangae">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListView" name="listView_TMlist"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_TMdescripte">
       <property name="title">
        <string>记忆库预览</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_TMname">
          <property name="text">
           <string>翻译记忆库名</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_TMnumber">
          <property name="text">
           <string>条目数量</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_TMnumber"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_TMcreatetime">
          <property name="text">
           <string>创建时间</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_TMcreatetime"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_TMlanguage">
          <property name="text">
           <string>语言对</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_TMlanguage"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_owner">
          <property name="text">
           <string>创建者</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="lineEdit_owner"/>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_TMname"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_createTM">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
